### ISSUE TITLE
Create edit a whole row insted a field

### ISSUE THREAD

**Issue by khanhdd123** [iamisti/mdDataTable#105]
How do I create a button for edit whole data of row instead a field??

**Comment by hoangta79** [comment]
Yes, this feature should support. I also expect to have this one.

**Comment by iamisti** [comment]
Let me think about this. It makes sense, but was not part of google material table concept. 
Since we can have multiple types of cells (dropdown in edit mode, input field in edit mode), we'd need to have the dropdown edit field first as a new feature, and then have a full implementation for this. But also possible in the other way around.

Would it be enough for you guys to have only text input fields for now?

**Comment by hoangta79** [comment]
Great. I think control supports edit row with text input fields is ok. With dropdown field, we can work around to include a dropdown when user click on textbox to select an item then bind text to the textbox. Thanks.
